### PR TITLE
[v6r15] Several minor fixes

### PR DESCRIPTION
**Pull request by atsareg**
VOMS2CSAgent - do not send mail if no changes performed
CSAPI.modifyUser() returns True or False if there were or not changes done
dirac-info - few more items to print out

### CODE REVIEW

File: ConfigurationSystem/Agent/UsersAndGroups.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def __syncCSWithVOMS( self ):
       for k in ( 'DN', 'CA', 'Email' ):
         csUserData[ k ] = ", ".join( csUserData[ k ] )
       result = csapi.modifyUser( user, csUserData, createIfNonExistant = True )
-      if not result[ 'OK' ]:
+      if not result[ 'OK' ] or not result['Value']:
         self.__adminMsgs[ 'Error' ].append( "Cannot modify user %s: %s" % ( user, result[ 'Message' ] ) )
```
Comment (fstagni) [inline ConfigurationSystem/Agent/UsersAndGroups.py]: If you enter here when result = {'OK': True, 'Value': ''} then accessing result['Message'] will issue a KeyError

File: __init__.py [hunk 2]
```diff
@@ -76,7 +76,7 @@
 # Define Version
 
 majorVersion = 6
-minorVersion = 13
+minorVersion = 15
```
Comment (fstagni) [inline __init__.py]: Fixed also in https://github.com/DIRACGrid/DIRAC/pull/2712 (commit: https://github.com/fstagni/DIRAC/commit/0a82caa2d8cbb386bdb4fc4a1f23751591fc6ca8)

File: Core/Utilities/Time.py [hunk 3]
```diff
@@ -264,10 +264,10 @@ def queryTime(f):
   """ Decorator to measure the function call time
   """
   def measureQueryTime(*args, **kwargs):
-    start = time.time()
+    start = nativetime.time()
     result = f(*args, **kwargs)
     if result['OK'] and not 'QueryTime' in result:
-      result['QueryTime'] = time.time() - start
+      result['QueryTime'] = nativetime.time() - start
```
Comment (fstagni) [inline Core/Utilities/Time.py]: Fixed also in https://github.com/DIRACGrid/DIRAC/pull/2712 (commit https://github.com/fstagni/DIRAC/commit/aa51b019631592f95d7b2f4024965e9d564f025b)

File: ConfigurationSystem/Agent/VOMS2CSAgent.py [hunk 4]
```diff
@@ -184,9 +187,12 @@ def __syncCSWithVOMS( self, vo ):
               groupsWithRole.append( group )
           userDict['Groups'] = list( set( groupsWithRole + [defaultVOGroup] ) )
           self.__adminMsgs[ 'Info' ].append( "Adding new user %s: %s" % ( newDiracName, str( userDict ) ) )
+          self.voChanged = True
           if self.autoAddUsers:
             self.log.info( "Adding new user %s: %s" % ( newDiracName, str( userDict ) ) )
-            self.csapi.modifyUser( newDiracName, userDict, createIfNonExistant = True )
+            result = self.csapi.modifyUser( newDiracName, userDict, createIfNonExistant = True )
+            if not result['OK']:
+              self.log.warn( 'Failed adding new user %s', newDiracName )
```
Comment (andresailer) [inline ConfigurationSystem/Agent/VOMS2CSAgent.py]: Is it intentional to not replace the %s ?

Comment (atsareg) [inline ConfigurationSystem/Agent/VOMS2CSAgent.py]: Fixed, thanks

**Comment by fstagni** [comment]
Review OK

**Pull request merged**